src/components/site/Images.md

```diff
@@ -0,0 +1,22 @@
+```jsx
+<Images
+  imageSrc="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+  content="Lorem Ipsum is simply dummy text of the printing and typesetting of Lorem Ipsum.industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five"
+></Images>
+```
+
+Html Snippet:
+```html
+<div>
+<div class="dg_image">
+  <img src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg" aria-hidden="true"></div>
```

Comment:
In this case, we actually don't want to use [aria-hidden](https://www.w3.org/WAI/PF/aria/states_and_properties#aria-hidden)="true" because this image is actually visible. We will also want to include an alt tag. This [link](https://www.commonplaces.com/blog/writing-alt-tags-for-images/) has a nice explanation of what the alt tag does.

Reply:
done